dev-games/rgbds/rgbds-0.9.4.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A free assembler/linker package for the Game Boy and Game Boy Color"
+DESCRIPTION="RGBDS\' four programs cover the whole compilation pipeline into a \
+ready-to-play ROM, from RGBGFX (the image converter) to RGBFIX (the header \
+\"fixer\")\.
+Each tool can easily be swapped for a custom one, or used stand-alone as part of another process "
```

Comment:
A space at the end instead of a full stop. Also, the lines above heed a 80 char limit, while this line fits in 100 char. I think we're on a 100 char limit in recipes (was 80 before), but to fit to the previous lines, let's stick to 80 chars per line. So:
```suggestion
Each tool can easily be swapped for a custom one, or used stand-alone as part \
of another process."
```